### PR TITLE
feat(events): add timezone tooltip. Clarifies HELP-968

### PR DESCRIPTION
**Pull request by mibaru-aegee**

### CODE REVIEW

File: src/views/events/Single.vue [hunk 1]
```diff
@@ -107,19 +107,19 @@
                   <td>{{ event.application_status | capitalize }}</td>
                 </tr>
                 <tr>
-                  <th>Application period starts</th>
+                  <th>Application period starts <tooltip text="Date and time in your timezone"></tooltip></th>
```
Comment (serge1peshcoff) [inline src/views/events/Single.vue]: Do you think it's possible to move it to a component? Then we can reuse it later, as it has the same text over and over.
Also I'd go with "The date and time are in your local timezone"

Comment (mibaru-aegee) [reply src/views/events/Single.vue]: Should I call the component "tooltip-timezone"?

Comment (serge1peshcoff) [reply src/views/events/Single.vue]: Looks nice to me.
For reference, you can check out `TimezoneNotification` component on event editing (in `src/views/events/Edit.vue`), I've done sth like this here

Comment (linuxbandit) [reply src/views/events/Single.vue]: I like this "encapsulation" :)

**Pull request merged**